arrays/practiceModular/main_test.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import "testing"
+
+func TestMonolithic(t *testing.T) {
+	area, peri := Monolithic(10, 5)
+
+	if area != 50 || peri != 30 {
```

Comment:
you can declare constants for 50 and 30 like const EXPECTED_AREA := 50 etc and use the same at multiple places